package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
Can you revert the changes on this file please

Reply:
Has been reverted